networkx/algorithms/bipartite/projection.py

```diff
@@ -337,10 +337,10 @@ def overlap_weighted_projected_graph(B, nodes, jaccard=True):
     >>> G = bipartite.overlap_weighted_projected_graph(B, [0, 2, 4])
     >>> print(G.nodes())
     [0, 2, 4]
-    >>> print(G.edges(data=True))
+    >>> print(list(G.edges(data=True)))
```

Comment:
I realize you just did a find/replace...but we don't need the print() statement. There are a number of these...